pisa_examples/resources/events/hs_test.csv

```diff
@@ -0,0 +1,4001 @@
+intercept,intercept_sigma,dom_eff,dom_eff_sigma,hole_ice_p0,hole_ice_p0_sigma,hole_ice_p1,hole_ice_p1_sigma,bulk_ice_abs,bulk_ice_abs_sigma,bulk_ice_scatter,bulk_ice_scatter_sigma,dm31,pid,reco_coszen,reco_energy
```

Comment:
What's in here, toy/dummy values? Suggest giving it a more descriptive filename (see the hdf5 events file in the same directory,toy, dummy, ...?) or adding a comment (shouldn't be a problem when parsed by pandas, https://pandas.pydata.org/docs/reference/api/pandas.read_csv.html).